### PR TITLE
including missing prototype

### PR DESCRIPTION
**Pull request by KrishMunot**

### CODE REVIEW

File: libredex/ReachableClasses.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 #include "ReachableClasses.h"
 
+#include <stdio.h>
```
Comment (bertmaher) [inline libredex/ReachableClasses.cpp]: What in this file needs stdio?  My compiler doesn't flag anything.

File: include/walkers.h [hunk 2]
```diff
@@ -27,7 +27,7 @@ void walk_methods(const T& scope, MethodWalkerFn walker) {
     for (auto vmethod : cls->get_vmethods()) {
       walker(vmethod);
     }
-  };
+  }
```
Comment (bertmaher) [inline include/walkers.h]: Can you revert these from this PR?  I agree it's kind of weird, but it's a really minor thing and not related to the described change.

Comment (KrishMunot) [inline include/walkers.h]: So, do I make a separate commit for this outside this PR?

File: libredex/ReachableClasses.cpp (hunk 1 above)
Comment (KrishMunot) [inline libredex/ReachableClasses.cpp]: The use of `%d` and `\n` made me trigger the most relevant header file for this.

Comment (bertmaher) [inline libredex/ReachableClasses.cpp]: I see.  The header this actually needs is Trace.h, not stdio.h.  It's transitively included by something else, but it's good practice anyways.  If you want to add it, add it in the block of relative includes.

File: include/walkers.h (hunk 2 above)
Comment (bertmaher) [inline include/walkers.h]: I'd prefer not to take this part of the change, it's extra overhead for my team to review trivial stuff.

**Comment by satnam6502** [comment]
It would be great if we could get rid of the use of `<stdio.h>` and use `<iostream>` instead consistently across the codebase.
